app/_how-tos/operator-konnect-plugin.md

```diff
@@ -0,0 +1,143 @@
+    url: /operator/reference/custom-resources/#kongplugin
+  - text: Plugins
+    url: /gateway/entities/plugin/
+tldr:
+  q: How do I enable and configure a plugin in {{site.konnect_short_name}} that's associated with another entity using KGO?
+  a: You can associate plugins with an entity, like a Consumer or Gateway Service, in {{site.konnect_short_name}}. To do this with KGO, you must create a `KongPlugin` and use `KongPluginBinding` to associate it with another entity.
+
+prereqs:
+  operator:
+    konnect:
+      auth: true
+      control_plane: true
+
+---
+
+## Create a `KongService` 
+ 
+In this tutorial, we'll bind a plugin to {{site.base_gateway}} entities, like a Gateway Service, using the `KongPluginBinding` CRD. 
+
+First, create a Gateway Service in [{{site.konnect_short_name}} Gateway Manager](/gateway-manager/) using the `KongService` CRD:
+
+<!-- vale off -->
+{% konnect_crd %}
+kind: KongService
+apiVersion: configuration.konghq.com/v1alpha1
+metadata:
+  name: example-service
+spec:
+  name: example-service
+  host: httpbin.konghq.com
+  protocol: http
+  controlPlaneRef:
+    type: konnectNamespacedRef
+    konnectNamespacedRef:
+      name: gateway-control-plane
+{% endkonnect_crd %}
+<!-- vale on -->
+
+## Create a `KongRoute`
+
+To expose the Service, create a `KongRoute` associated with the `KongService` defined previously:
+
+<!-- vale off -->
+{% konnect_crd %}
+kind: KongRoute
+apiVersion: configuration.konghq.com/v1alpha1
+metadata:
+  name: example-route
+spec:
+  name: example-route
+  protocols:
+  - http
+  paths:
+  - /anything
+  serviceRef:
+    type: namespacedRef
+    namespacedRef:
+      name: example-service
+{% endkonnect_crd %}
+<!-- vale on -->
+
+## Enable a `KongPlugin` and create a `KongPluginBinding`
+
+In this tutorial, you'll enable a simple configuration of the [Rate Limiting](/plugins/rate-limiting/) plugin. 
+
+First, enable the plugin:
+
+<!-- vale off -->
+{% konnect_crd %}
+kind: KongPlugin
+apiVersion: configuration.konghq.com/v1
+metadata:
+  namespace: kong
+  name: rate-limiting-minute-5
+plugin: rate-limiting
+config:
+  policy: local
+  minute: 5
+  hour: 1000
+{% endkonnect_crd %}
+<!-- vale on -->
+
+Then, to bind the plugin to the Service, create a `KongPluginBinding`:
+
+<!-- vale off -->
+{% konnect_crd %}
+kind: KongPluginBinding
+apiVersion: configuration.konghq.com/v1alpha1
+metadata:
+  name: binding-service-example-rate-limiting
+spec:
+  pluginRef:
+    kind: KongPlugin
+    name: rate-limiting-minute-5
+  targets:
+    serviceRef:
+      group: configuration.konghq.com
+      kind: KongService
+      name: example-service
+  controlPlaneRef:
+    type: konnectNamespacedRef
+    konnectNamespacedRef:
+      name: gateway-control-plane
+{% endkonnect_crd %}
+<!-- vale on -->
+
+## Validate
+
+<!-- vale off -->
+{% validation kubernetes-resource %}
+kind: KongPluginBinding
+name: binding-service-example-rate-limiting
+{% endvalidation %}
+<!-- vale on -->
```

Comment:
I'm not getting the expected result, but I don't know if it's because of the issue I had in the previous step